odoo/addons/base/views/res_partner_views.xml

```diff
@@ -138,7 +138,7 @@
                                 <field name="zip" placeholder="ZIP" class="o_address_zip"/>
                                 <field name="country_id" placeholder="Country" class="o_address_country" options='{"no_open": True, "no_create": True}'/>
                             </div>
-                            <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
+                            <field name="website" string="Website" widget="url" placeholder="e.g. https://www.odoo.com"/>
```

Comment:
what's the string of field website if it's not already "Website" ? :suspect:

Reply:
Because `website_id` already has the same string and that would raise a warning
https://github.com/odoo/odoo/blob/3fa3676240b48cce049d79f20292ac1eea8dd727/addons/website/models/res_partner.py#L8-L10

https://github.com/odoo/odoo/blob/3fa3676240b48cce049d79f20292ac1eea8dd727/addons/website/models/mixins.py#L218-L222

https://github.com/odoo/odoo/blob/3fa3676240b48cce049d79f20292ac1eea8dd727/addons/website/models/mixins.py#L133-L144